app/src/main/java/com/vero/tasky/auth/util/PasswordValidator.kt

```diff
@@ -3,28 +3,26 @@ package com.vero.tasky.auth.util
 import com.vero.tasky.R
 import com.vero.tasky.auth.domain.usecase.password.PasswordValidationResult
 
-object PasswordErrorHandler {
+object PasswordValidator {
 
-    fun onPasswordValid(
+    fun validatePassword(
```

Comment:
And this method doesnt really validate the password

So id name it
.parse, .handle, or similar (depending on the name you decide to use for the class)